.mage/docs.go

```diff
@@ -0,0 +1,87 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ttnmage
+
+import (
+	"github.com/magefile/mage/mg"
+	"github.com/magefile/mage/sh"
+	"io/ioutil"
+	"os"
+	"text/template"
+)
+
+type HugoConfig struct {
+	CurrentVersion string
+}
+
+// Docs namespace
+type Docs mg.Namespace
+
+var hugoArgs = []string{"run", "github.com/gohugoio/hugo", "-s", "doc"}
```

Comment:
I'd prefer a function/method like `execHugo`, which would pass these to the `execGo`:
```suggestion
func execHugo(args ...string) error {
	return execGo("run", append([]string{"github.com/gohugoio/hugo", "-s", "doc"}, args...))
}
```